source/ui/BioXAS/BioXASSIS3820ScalerChannelView.h

```diff
@@ -0,0 +1,58 @@
+#ifndef BIOXASSIS3820SCALERCHANNELVIEW_H
+#define BIOXASSIS3820SCALERCHANNELVIEW_H
+
+#include <QWidget>
+#include <QLayout>
+
+#include "beamline/CLS/CLSSIS3820Scaler.h"
+#include "ui/CLS/CLSSIS3820ScalerView.h"
+
+class AMExtendedControlEditor;
+
+class BioXASSIS3820ScalerChannelView : public CLSSIS3820ScalerChannelView
+{
+    Q_OBJECT
+
+public:
+	/// Constructor.
+	explicit BioXASSIS3820ScalerChannelView(CLSSIS3820ScalerChannel *channel, bool biasEnabledShown = false, bool biasShown = false, bool darkCurrentShown = false, QWidget *parent = 0);
+	/// Destructor.
+	virtual ~BioXASSIS3820ScalerChannelView();
+	/// Returns the channel being viewed.
+	CLSSIS3820ScalerChannel* channel() const { return channel_; }
+	/// Returns whether the bias enabled editor is shown.
+	bool biasEnabledShown() const { return biasEnabledShown_; }
+	/// Returns whether the bias editor is shown.
+	bool biasShown() const { return biasShown_; }
+	/// Returns whether the dark current is shown.
+	bool darkCurrentShown() const { return darkCurrentShown_; }
+
+public slots:
+	/// Sets whether the bias enabled editor is shown.
+	void setBiasEnabledEditorShown(bool shown);
```

Comment:
Okay, okay.  Broken record, I know.  To be consistent with existing code and the Qt API, using visible, show, and hide would be preferable.